ui-tests/src/test/java/io/syndesis/qe/steps/integrations/datamapper/DataMapperSteps.java

```diff
@@ -156,4 +156,8 @@ public void separatePresentFielsIntoTwo(String input, String output1, String fir
 
     }
 
+    @When("^defined constant \"([^\"]*)\" of type \"([^\"]*)\" in data mapper$")
```

Comment:
This does not sound idiomatic (neither of the two idioms we currently use). Perhaps "define constant ..." ?